docs/book/user-guide/tutorials/managing-scheduled-pipelines.md

```diff
@@ -137,7 +139,7 @@ zenml pipeline schedule list --pipeline_id my_pipeline_id
 
 Here's an example of what the CLI output might look like:
 
-![Output of `zenml pipeline schedule list`](../../.gitbook/assets/pipeline-schedules-list.png)
+![Output of zenml pipeline schedule list](../../.gitbook/assets/pipeline-schedules-list.png)
```

Comment:
```suggestion
![Output of zenml pipeline schedule list](../.gitbook/assets/pipeline-schedules-list.png)
```

And could you move the png file from the root assets folder to the assets folder in the user-guide subfolder? I messed that up, but I doubt that's what's causing the issues.